stock_warehouse_orderpoint_stock_info/tests/test_stock_warehouse_orderpoint.py

```diff
@@ -1,66 +1,51 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 OdooMRP Team
 # Copyright 2016 AvanzOSC
 # Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
 # Copyright 2016 Serpent Consulting Services Pvt. Ltd.
 # Copyright 2016-17 Eficent Business and IT Consulting Services, S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests import common
+from odoo.tests.common import SavepointCase
 
 
-class TestStockWarehouseOrderpoint(common.TransactionCase):
+class TestStockWarehouseOrderpoint(SavepointCase):
```

Comment:
Just wondering why you are moving to Savepointcase?

Reply:
Faster than TransactionCase as the setup can be done only once for all tests (here there's only 1 test but if a test is added, direct benefit)

Reply:
Fair enough! thanks